pkg/controller/container-runtime-config/helpers.go

```diff
@@ -539,24 +539,93 @@ func validateRegistriesConfScopes(insecure, blocked, allowed []string, icspRules
 		}
 	}
 
-	for _, icsp := range icspRules {
-		for _, mirrorSet := range icsp.Spec.RepositoryDigestMirrors {
+	idmsRuleSourcePolicy := map[string]apicfgv1.MirrorSourcePolicy{}
+	itmsRuleSourcePolicy := map[string]apicfgv1.MirrorSourcePolicy{}
```

Comment:
If it is desirable to reject conflicts, shouldn’t it reject also conflicts between IDMS/ITMS?